containers/image.go

```diff
@@ -5,34 +5,24 @@ import (
 	"fmt"
 	"log"
 	"os"
+	"rkd/helpers"
 	"strings"
 
 	"github.com/containers/image/v5/copy"
 	"github.com/containers/image/v5/docker/archive"
 	"github.com/containers/image/v5/docker/reference"
+	"github.com/containers/image/v5/manifest"
 	"github.com/containers/image/v5/signature"
 	"github.com/containers/image/v5/transports/alltransports"
 	"github.com/containers/image/v5/types"
+	"github.com/opencontainers/go-digest"
 )
 
 // DownloadImage download docker images from src and create docker-archive
-func DownloadImage(imgList []string, dest string) error {
-
-	// Contexts
-	defaultPolicy, err := signature.NewPolicyFromFile("policy.json")
-	if err != nil {
-		log.Printf("Default policy err.\n")
-		return err
-	}
-	policyContext, err := signature.NewPolicyContext(defaultPolicy)
-	if err != nil {
-		log.Printf("Policy context err.\n")
-		return err
-	}
-	defer policyContext.Destroy()
+func DownloadImage(imgList []string, dest string, fetchSignature bool) error {
 
 	// Create new dest archive
-	aw, err := archive.NewWriter(nil, dest)
+	aw, err := archive.NewWriter(nil, fmt.Sprintf("%s/images.tar", dest))
```

Comment:
`dest` replaced by `destImg` in `cmd/download.go` so do not provide the destination filename here juste use `aw, err := archive.NewWriter(nil, dest)`